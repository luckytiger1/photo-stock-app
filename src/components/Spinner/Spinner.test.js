import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import Spinner from './Spinner';

it('<SearchBox /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<Spinner />)).toMatchSnapshot();
});
