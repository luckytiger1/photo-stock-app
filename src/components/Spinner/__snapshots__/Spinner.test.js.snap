// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> renders correctly 1`] = `
<div
  className="d-flex justify-content-center"
>
  <div
    className="loadingio-spinner-spinner-t1brovegdl"
  >
    <div
      className="ldio-y4i1u5rbs4"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
